salesforce/management/commands/update_faculty_status.py

```diff
@@ -3,23 +3,31 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.models import Group
 from django.contrib.auth.models import User, Group
-
+from social.apps.django_app.default.models import DjangoStorage as SocialAuthStorage
 
 class Command(BaseCommand):
     help = "Add user to faculty group if confirmed by salesforce"
 
     def add_arguments(self, parser):
-        parser.add_argument('cms_id', type=int)
-        parser.add_argument('accounts_id', type=int)
-
+        parser.add_argument('cms_id', nargs='?', type=int,default=None)
+        parser.add_argument('--all', action='store_true', default=False)
     def handle(self, *args, **options):
+        if options['all']:
+            accounts_id = None
+        else:
+            social_user = SocialAuthStorage.user.objects.filter(user_id = options['cms_id'])
```

Comment:
Any reason you are calling this cms_id? I thought it wanted the ID of the CMS/Site, maybe call it user_id for clarity?